packages/botonic-react/tests/utils.test.js

```diff
@@ -26,3 +29,100 @@ describe('Regex serialization / deserialization', () => {
     })
   })
 })
+
+jest.mock('../src/util/environment', () => ({
+  ...jest.requireActual('../src/util/environment'),
+  isURL: jest.fn(),
+  normalize: jest.fn(),
+}))
+
+const createScript = src => {
+  const script = document.createElement('script')
+  script.src = src
+  document.head.appendChild(script)
+  return script
+}
+
+const removeScript = script => {
+  document.head.removeChild(script)
+}
+
+describe('staticAsset function', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  test('should return the path if it is a URL', () => {
+    //arrange
+    isURL.mockReturnValue(true)
```

Comment:
why mocking isURL? in this case it is a url in fact, right?

Reply:
Mocking it ensures that what I am testing is the behaviour of the `staticAsset` function without relying on the correctness of `isURL` method. In fact, the isURL function should have a unit test itself.

Reply:
yes, this was exactly what I meant..